test/functional/core/path_spec.lua

```diff
@@ -129,4 +129,48 @@ describe('file search (gf, <cfile>)', function()
     test_cfile([[\\.\UNC\LOCALHOST\c$\temp\test-file.txt]], [[.]], [[\\.\UNC\LOCALHOST\c$\temp\test-file.txt]])
     test_cfile([[\\127.0.0.1\c$\temp\test-file.txt]], [[127.0.0.1]], [[\\127.0.0.1\c$\temp\test-file.txt]])
   end)
+
+  describe('using `finddir`', function()
```

Comment:
backticks are not useful in a test name, it's more visually useful to use parens to indicate a function
```suggestion
  describe('using finddir()', function()
```